Zend/zend_exceptions.c

```diff
@@ -144,12 +144,12 @@ void zend_exception_restore(void) /* {{{ */
 }
 /* }}} */
 
-ZEND_API ZEND_COLD void zend_throw_exception_internal(zval *exception) /* {{{ */
+ZEND_API ZEND_COLD void zend_throw_exception_internal_obj(zend_object *exception) /* {{{ */
```

Comment:
Given how rarely used this API is, I think it would be better to just change the argument type.

Reply:
OK, I can do that!